great-idea/index.html

```diff
@@ -16,7 +16,98 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    
+
+      <div>
+        <nav class="top-nav">
+          <a href="file:///Users/rayfairley/GitProjects/UI-III-Flexbox/great-idea/services.html">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+      
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </nav>
+      </div>
+      
+      <section class="header">
```

Comment:
Small detail, but you could've went with a `<header>` instead and targeted that element in CSS. It's more semantic.